validator/inliner/inliner.go

```diff
@@ -2,24 +2,26 @@ package main
 
 import (
 	"bytes"
+	"fmt"
 	"io/ioutil"
-	"strconv"
 
 	"golang.org/x/tools/imports"
 )
 
 func main() {
 	out := bytes.Buffer{}
 	out.WriteString("package validator\n\n")
-	out.WriteString(`var Prelude = &ast.Source{Name: "prelude.graphql", Input: `)
 
 	file, err := ioutil.ReadFile("prelude.graphql")
 	if err != nil {
 		panic(err)
 	}
 
-	out.WriteString(strconv.Quote(string(file)))
-	out.WriteString("}\n")
+	fmt.Fprintf(&out, `var Prelude = &ast.Source{
+		Name: "prelude.graphql",
+		Input: %q,
+		BuiltIn: true,
+	}`, string(file))
```

Comment:
The original code doesn't have a `BuiltIn: true` code. To enable introspection query, we have to add this field. (But `validator/prelude.go` has this field I don't know why 🤔 )